### ISSUE TITLE
refactoring_sandbox : Hydra : Moved USD Proxy makes Prim selection highlight still behave as if at origin

### ISSUE THREAD

**Issue by BigRoy** [Autodesk/maya-usd#82]
### Issue

When moving the USD Proxy Shape's transform around the Prim UFE Selection still selects as if the object was not transformed from the origin.

This is only noticeable in Hydra viewport as Viewport 2.0 does not support the transformation of the USD scene through the Maya transform currently, see: #71 

Potentially this is fixed once VP2 also supports the transformation with a fix for #71.

#### To reproduce

1. Create a proxy shape and load USD file.
2. Set viewport to Hydra.
3. Move the proxy shape transform.
4. Now select prims in the viewport, they will misalign visually because the selection will behave as if you didn't move the transform in Step 3.

### Specs

- Windows 10
- Maya 2019.2
- refactoring_sandbox ( [`d0bb624`](https://github.com/Autodesk/maya-usd/commit/caa921c1cb3b59ea6acc84540ebea9a05b1fedf9) ) 
- Pixar USD dev ( `b85ddac2` )

**Comment by HdC-adsk** [comment]
Hi @BigRoy Thanks for raising this issue. I am looking into it.

**Comment by HdC-adsk** [comment]
Logged as MAYA-99904 internally.

**Comment by cfmoore007** [comment]
@BigRoy  - my tests look good here.  Can you confirm the same ?

**Comment by BigRoy** [comment]
Unfortunately I am out of office this week (at Unreal Academy in London) and will be unable to confirm the fixes/changes of #112 until early next week.

**Comment by HdC-adsk** [comment]
The issue has been fixed by #112 in the dev branch, but it cannot be validated at the moment because the mtoh plugin is not merged to dev branch yet.

**Comment by BigRoy** [comment]
@HdC-adsk any ETA on when that might happen and have these fixes propagated to `refactoring_sandbox` (or the other way around)?

**Comment by kxl-adsk** [comment]
@BigRoy Regarding merging remaining work from refactoring_sandbox - we have always said that this branch will have a short lifetime and it's a priority for us to be all together in the dev branch.
As for when fixes will be propagated to refactoring_sandbox - we will be doing them often, just didn't get to it yet. FWIW. It shouldn't be blocking for you, the dev branch should merge cleanly into refactoring_sandbox if you want to try it locally.

**Comment by BigRoy** [comment]
Great. I'll get to using the `dev` branch because if I understand you correctly that's the branch that will be the "end goal" at some point. 

I can confirm this works as expected in the latest `dev` branch in Viewport 2.0. As @HdC-adsk mentioned, the `dev` branch does not have the `mtoh` viewport renderer, as such any potential mismatch in selection can't be tested there.

For now I'll assume this will also be fixed once the `mtoh` is merged into `dev`. Feel free to reopen if the issue reappears.

**Comment by kxl-adsk** [comment]
The merge is in https://github.com/Autodesk/maya-usd/pull/120
